app_dart/lib/src/foundation/github_checks_util.dart

```diff
@@ -70,11 +73,35 @@ class GithubChecksUtil {
   }
 
   Future<github.CheckRun> createCheckRun(
-    github.GitHub gitHubClient,
+    Config cocoonConfig,
+    github.RepositorySlug slug,
+    String name,
+    String headSha,
+  ) async {
+    const RetryOptions r = RetryOptions(
+      maxAttempts: 3,
+      delayFactor: Duration(seconds: 2),
+    );
```

Comment:
With new token generated, do we still need the retry?

Reply:
Yes, even with the token freshly generated sometimes github is still rejecting the auth.